src/core/search/indices.cc

```diff
@@ -8,12 +8,15 @@
 #include <absl/strings/ascii.h>
 #include <absl/strings/numbers.h>
 #include <absl/strings/str_split.h>
-#include <unicode/brkiter.h>
-#include <unicode/unistr.h>
+
+#define UNI_ALGO_STATIC_DATA
+#define UNI_ALGO_DISABLE_NFKC_NFKD
+
+#include <uni_algo/case.h>
+#include <uni_algo/ranges_word.h>
```

Comment:
We use add_third_party to include deps and use the headers directly ignoring its cmake so the defines need to be placed here.